src/index.tsx

```diff
@@ -1392,9 +1388,70 @@ class App extends React.Component<{}, AppState> {
     this.setState({ currentItemBackgroundColor: color });
   };
 
+  private updateElement = (element: any, values: any) => {
+    /** Use of Object.assign instead spread because we want to keep ref to element for update is value in initial object */
+    Object.assign(element, values);
```

Comment:
Have you tested undo action? Mutating original element can be dangerous here

Reply:
Undo action exist ? Cmd+Z ? I don’t see it

Reply:
Oh my PR was launch before your undo/redo was merge, sorry. I rebase on master